src/main/webapp/comments.html

```diff
@@ -3,65 +3,90 @@
   <head>
     <meta charset="UTF-8" />
     <title>Capstone</title>
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="styleComments.css" />
```

Comment:
You all should probably get together and decide if you're using lowerCamelCase or dash-case for names like this. Just be consistent, I think Nina's using dash-case.

Reply:
I will use dash case then :)

Reply:
Saw that Dagmawi is also using camel so I think we will get together and figure that out soon.